tests/all/bindgen.rs

```diff
@@ -4,12 +4,18 @@ use wasm_pack::bindgen;
 #[test]
 #[cfg(any(
     all(target_os = "linux", target_arch = "x86_64"),
-    all(target_os = "macos", target_arch = "x86_64")
+    all(target_os = "macos", target_arch = "x86_64"),
+    all(windows, target_arch = "x86_64"),
 ))]
 fn can_download_prebuilt_wasm_bindgen() {
+    use std::env;
     let dir = tempfile::TempDir::new().unwrap();
-    bindgen::download_prebuilt_wasm_bindgen(dir.path(), "0.2.19").unwrap();
-    assert!(dir.path().join("bin").join("wasm-bindgen").is_file());
+    bindgen::download_prebuilt_wasm_bindgen(dir.path(), "0.2.21").unwrap();
```

Comment:
curious about updating the version number here- that's maintenance we aren't going to want to be doing...

Reply:
Oh this is just because the 0.2.19 release didn't have a windows release due to a bug in wasm-bindgen's configuration, but 0.2.21 has it fixed. This shouldn't need to change again.

Reply:
0.2.19 hit a bug hence the number update. this won't need to be maintained. resolved.